### PR TITLE
Add new rules to Checkstyle-Operator wrap

### PR DESCRIPTION
**Pull request by JosephineKwa**
Added operator wrap rule.
Checks that assignment operator `=` is on the end of line rather than on a new line.
Checks that all other operators (applicable to the operator wrap rule) are on a new line rather than on the end of line

### CODE REVIEW

File: src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java [hunk 1]
```diff
@@ -270,8 +270,9 @@ public boolean isResponseCommentVisibleForUser(String userEmail, String courseId
         }
         
         boolean isVisibilityFollowingFeedbackQuestion = relatedComment.isVisibilityFollowingFeedbackQuestion;
-        boolean isVisibleToGiver = isVisibilityFollowingFeedbackQuestion ? 
-                true : relatedComment.isVisibleTo(FeedbackParticipantType.GIVER);
+        boolean isVisibleToGiver = isVisibilityFollowingFeedbackQuestion 
+                                 ? true 
+                                 : relatedComment.isVisibleTo(FeedbackParticipantType.GIVER);
```
Comment (wkurniawan07) [inline src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java]: `condition1 ? true : condition2` is equivalent to `condition1 || condition2`.

**Comment by wkurniawan07** [comment]
Looks good, other than the merge conflict and the one I pointed out.

**Comment by wkurniawan07** [comment]
There's a conflict again, apologies beforehand if I'm the one who caused it :p

**Comment by JosephineKwa** [comment]
Merge conflict resolved!

**Comment by wkurniawan07** [comment]
Looks good, better merge soon before it's conflicting again :p

**Pull request merged**